frontend/src/toolbar/toolbarLogic.ts

```diff
@@ -26,6 +26,14 @@ export const toolbarLogic = kea<toolbarLogicType>({
         actionId: [props.actionId || null, { logout: () => null, clearUserIntent: () => null }],
         userIntent: [props.userIntent || null, { logout: () => null, clearUserIntent: () => null }],
         buttonVisible: [true, { showButton: () => true, hideButton: () => false, logout: () => false }],
+        dataAttributes: [
+            ((typeof props.dataAttributes === 'string'
```

Comment:
Q: Why is this needed? (I see a similar joining in the backend, but this is weird)

Also, Perhaps a cleaner way to do this is with a selecror rather than reducer.

Reply:
Indeed, was not needed.